be/src/agent/agent_task.cpp

```diff
@@ -286,6 +289,18 @@ void run_alter_tablet_task(const std::shared_ptr<AlterTabletAgentTaskRequest>& a
         if (task_type == TTaskType::ALTER) {
             alter_tablet(agent_task_req->task_req, signatrue, &finish_task_request);
         }
+        finish_task(finish_task_request);
+    } else {
+        // Response should be reported to FE even if timeout.
+        TFinishTaskRequest finish_task_request;
+        finish_task_request.__set_backend(BackendOptions::get_localBackend());
+        finish_task_request.__set_task_type(agent_task_req->task_type);
+        finish_task_request.__set_signature(agent_task_req->signature);
+        TStatus task_status;
+        task_status.__set_status_code(TStatusCode::CANCELLED);
```

Comment:
there is a `TStatusCode::TIMEOUT`, should it be timed out or cancelled?

Reply:
ok